src/test/validation_chainstatemanager_tests.cpp

```diff
@@ -349,7 +351,7 @@ struct SnapshotTestSetup : TestChain100Setup {
         }
 
         // Snapshot should refuse to load after one has already loaded.
-        BOOST_REQUIRE(!CreateAndActivateUTXOSnapshot(this));
+        //BOOST_REQUIRE(!CreateAndActivateUTXOSnapshot(this));
```

Comment:
In 99e28b1a58b9864e4f393a2a065ef63f12eb8b85 "Tighten requirements for adding elements to setBlockIndexCandidates"

Is this check expected to fail? I tried uncommenting it and the test passes.